docs/advanced-guide/overriding-default/page.md

```diff
@@ -0,0 +1,73 @@
+# Overriding Default
+
+GoFr allows to override default behavior of its features.
+
+## Raw response format
+
+GoFr by default wraps a handler's return value and assigns it to the "data" field in a response.
+
+### Example
+
+```go
+package main
+
+import "gofr.dev/pkg/gofr"
+
+type user struct {
+  Id   int    `json:"id"`
```

Comment:
Id field should be refactored to ID, as per effective go guidelines, if in code linter would have thrown an error.

While, JSON tag id remains as it is

Reply:
Done.